css/style.css

```
@@ -16,4 +17,161 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+ /* header section */
+ header{
+     display: flex;
+     justify-content: space-evenly;
+     align-content: center;
+ }
 
+ header img{
+     height: 2.5em;
+     padding: 1rem ;
+   
+}
+
+/* Nav bar */
+
+
+ nav ul{
+     display: flex;
+     list-style: none;
+     justify-content: space-evenly;
+     width: 50rem;
+     padding: 1rem 0rem;
+
+ }
+
+ nav ul a{
+     text-decoration: none;
+     font-size: 1.5rem;
+     color:rgb(197, 189, 189);
+ }
+
+nav ul a:hover{
+    color:rgb(240, 104, 13);
+}
+
+li:first-of-type a{
+    color: rgb(105, 102, 102);
+    font-weight: 500;
+}
+x-special/nautilus-clipboard
+
+/*Main content*/
+.hero{
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+
+}
+
+.hero img{
+    display: flex;
+    justify-content: center;
+    padding-left: 1rem;
+    width: 102vh;
+    height: 100%;
+}
+
+
+.hero-text{
+    color: white;
+    position: absolute;
+    top: 30%;
+    left: 47%;
+    transform: translate(-50%, -50%);
+    text-align: center;
+}
```

```suggestion
.hero{
    color: white;
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    text-align: center;
}
```
If we delete the `<div class="hero-text">` in the HTML like my other suggestion, we can use flexbox to align the text and button in the centre which is a better way than using `position: absolute`